### ISSUE TITLE
Java Backend Outdated?

### ISSUE THREAD

**Issue by potatoelixir** [Kode/Kha#1162]
Is it just me, or is Java backend outdated?
I can't get it to compile; first, it complained about Int32Array missing, so I implemented that myself. Now, it's complaining about SystemImpl.login, SystemImpl.automaticSafeZone, SystemImpl.setSafeZone, SystemImpl.unlockAchievement, and Font.widthOfCharacters.

If this is something that needs to be fixed, I think I can take some time to first add stubs for everything that's missing, implement whatever functionality that urgently needs to be implemented, then create a PR. Please let me know.

**Comment by RblSb** [comment]
Yes, this backend makes little sense to support, it designed to have only g2 support with separated implementation on swing api. I updated it once some months ago, but only one useful idea is can be fast haxe sys api tests on it (like canvas-based file manager).

**Comment by potatoelixir** [comment]
Ok, so it's just not maintained. It's not like I'm gonna use Java backend, and it seems no one else is either, so won't waste my time trying to fix it up. Do you think it'd be worth it to leave this issue open, just in case someone else tries to compile to Java target and it doesn't work?

**Comment by RblSb** [comment]
Will see what Robert says, maybe he want to remove this backend at all.

**Comment by RobDangerous** [comment]
Yes, keep the issue open. The Java backend, like the WPF backend, makes sense for integrating Kha components into Java programs but AFAIK currently nobody is using it.

**Comment by sangohan** [comment]
it would be interesting to be able to use the java backend for android rather than CPP backend

**Comment by RblSb** [comment]
Android-java has performance problems like https://github.com/Kode/Kha/issues/1051 and something about audio support, so it's pretty useless. You can checkout android-hl for faster debug builds from scratch.
